### DISCUSSION TITLE
How to use Pest with this package?

### DISCUSSION THREAD

**Discussion by pramod-praella** [Kyon147/laravel-shopify#190] [General]
Hello ALL!
Is there anyone who knows how to use Pest with this package?
I tried to implement the test feature with Pest but was not able to run it under the verifyShopify middleware.

**Comment by Kyon147** [comment]
Hi @pramod-praella 

Pest is the testing framework correct? 

You'll have to create a fake user for your test and log them in, it will require a bit of work on your part as that middleware relies on the Shopify auth stuff. So you#ll have to mock up all that process for your testing framework.

**Reply by pramod-praella** [reply]
Hello @Kyon147 ,
I am trying the same but cannot authenticate the user with verifyShopify middleware.
Please let me know how to authenticate the user for direct requests.
